src/main/java/gov/cabinetofice/gapuserservice/web/SuperAdminController.java

```diff
@@ -27,22 +35,65 @@ public class SuperAdminController {
     private final RoleService roleService;
     private final OneLoginUserService oneLoginUserService;
 
-    @GetMapping("/super-admin-dashboard")
-    public ResponseEntity<SuperAdminDashboardPageDto> superAdminDashboard(final HttpServletRequest httpRequest, final Pageable pagination) {
+    @GetMapping("/super-admin-dashboard") ResponseEntity<?> superAdminDashboard(
+            final HttpServletRequest httpRequest,
+            final Pageable pagination,
+            @RequestParam(value = "departments", required = false) String departments,
+            @RequestParam(value = "roles", required = false) String roles,
```

Comment:
Should be possible to pass these in as List<String>, or String[]

Reply:
https://stackoverflow.com/questions/4596351/binding-a-list-in-requestparam#:~:text=myparam%3DmyValue1%26myparam%3DmyValue2%26myparam%3DmyValue3